src/paperqa/readers.py

```diff
@@ -445,6 +495,18 @@ async def read_doc(  # noqa: PLR0912
                 f"|size={chunk_chars}|overlap={overlap}{enrichment_summary}"
             ),
         )
+    elif str_path.endswith((".docx", ".xlsx", ".pptx")):
+        chunked_text = chunk_pdf(
+            parsed_text, doc, chunk_chars=chunk_chars, overlap=overlap
+        )
+        chunk_metadata = ChunkMetadata(
+            size=chunk_chars,
+            overlap=overlap,
+            name=(
+                f"paper-qa={pqa_version}|algorithm=overlap-office"
+                f"|size={chunk_chars}|overlap={overlap}{enrichment_summary}"
+            ),
+        )
```

Comment:
Since this matches the `.pdf` branch, do you mind consolidating the two of them, and making the `algorithm=overlap` to be general